content/en/agent/kubernetes/control_plane.md

```diff
@@ -306,7 +308,28 @@ scheduler:
     bind-address: 0.0.0.0
 ```
 
+## Kubernetes on Amazon Elastic Kubernetes Service {#EKS}
+
+On EKS, AWS exposes API Server metrics via [a single endpoint][5]. This allows the Datadog Agent to obtain API Server metrics using endpoint checks as described in the [Kubernetes API Server metrics check documentation][6]. To conigure the check, add the following annotations to the `default/kubernetes` service:
```

Comment:
```suggestion
On Elastic Kubernetes Service (EKS), AWS exposes API server metrics on a [single endpoint][5]. This allows the Datadog Agent to obtain API server metrics using endpoint checks as described in the [Kubernetes API Server metrics documentation][6]. To configure the check, add the following annotations to the `default/kubernetes` service:
```

Reply:
Thanks @ruthnaebeck ! I noticed we capitalized `Server` in `API Server` elsewhere in the doc, I'll update the capitalization in other sections too. This seems to be consistent with what we do elsewhere (and in the Kubernetes docs)